autoload/airline/extensions/branch.vim

```diff
@@ -132,6 +135,7 @@ function! s:display_git_branch()
     endif
   catch
   endtry
+  call airline#util#suspend_focusgain(0)
```

Comment:
This won't work because we need the ignore to last beyond this function. FocusGained hasn't occurred yet. It won't happen until vim does its next tick.

This PR is equivalent to chrisbra's original fix in f4795532c6a2a4f65ed7464ba5abde2f0e27e555.